go.mod

```diff
@@ -186,5 +191,13 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 	gorm.io/driver/sqlite v1.2.4 // indirect
-	gotest.tools/v3 v3.0.3 // indirect
+	modernc.org/cc/v3 v3.32.4 // indirect
+	modernc.org/ccgo/v3 v3.9.2 // indirect
+	modernc.org/libc v1.9.5 // indirect
+	modernc.org/mathutil v1.2.2 // indirect
+	modernc.org/memory v1.0.4 // indirect
+	modernc.org/opt v0.1.1 // indirect
+	modernc.org/sqlite v1.10.6 // indirect
```

Comment:
Do we have a plan to migrate the rest of the codebase to this CGO-free driver too?

Reply:
nope!  it's just an indirect dependency which a separate repo uses.